src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -171,6 +171,7 @@ public partial class FluentDataGrid<TGridItem> : FluentComponentBase, IHandleEve
     private int? _lastRefreshedPaginationStateHash;
     private object? _lastAssignedItemsOrProvider;
     private CancellationTokenSource? _pendingDataLoadCancellationTokenSource;
+    private bool _manualGrid;
```

Comment:
Do you think better to use a private property "auto-calulcated"?
`private bool ManualGrid => !_columns.Any();`

Reply:
No, this doesn't work as the _columns object is not populated yet at creation time.